src/index.html

```diff
@@ -1,13 +1,39 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img src="images/imac.jpeg" class="card__image"
+      alt="imac">
+      <div class="card__item">
+        <h1 class="card__item-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
+        <p class="card__item-code">Код товара: 195434</p>
+        <div class="card__item-rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__item-rated">Отзывов: 5</p>
+        </div>
+        <div class="card__item-value">
+          <p class="card__item-value-text">Цена:</p>
+          <p class="card__item-value-number">69 999 грн</p>
+        </div>
```

Comment:
це не є окремий блок по БЕМ
```suggestion
        <div class="card__price">
          <p class="card__price-text">Цена:</p>
          <p class="card__price-value">69 999 грн</p>
        </div>
```